lab_3_ann_retriever/main.py

```diff
@@ -149,6 +181,15 @@ def __init__(self, corpus: list[list[str]]) -> None:
         Args:
             corpus (list[list[str]]): Tokenized documents to vectorize
         """
+        if not (isinstance(corpus, list) and all(isinstance(tokens, list)
```

Comment:
is it really neede to check types in `__init__`?

Reply:
I guess no...